2021-07-22 10:54:51 =ERROR REPORT====
Error in process <0.269.0> with exit value:
{#{'__exception__' => true,'__struct__' => 'Elixir.Protocol.UndefinedError',description => <<>>,protocol => 'Elixir.String.Chars',value => <0.269.0>},[{'Elixir.String.Chars','impl_for!',1,[{file,"lib/string/chars.ex"},{line,3}]},{'Elixir.String.Chars',to_string,1,[{file,"lib/string/chars.ex"},{line,22}]},{'Elixir.Processor',kill_process,1,[{file,"lib/processor.ex"},{line,16}]}]}
2021-07-22 11:10:59 =CRASH REPORT====
  crasher:
    initial call: application_master:init/4
    pid: <0.271.0>
    registered_name: []
    exception exit: {{bad_return,{{'Elixir.Processor.Application',start,[normal,[]]},{'EXIT',{undef,[{'Elixir.Supervisor',start_link,[[{strategy,one_for_one},{name,'Elixir.Processor.Supervisor'}]],[]},{'Elixir.Processor.Application',start,2,[{file,"lib/processor/application.ex"},{line,10}]},{application_master,start_it_old,4,[{file,"application_master.erl"},{line,293}]}]}}}},[{application_master,init,4,[{file,"application_master.erl"},{line,142}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,226}]}]}
    ancestors: [<0.270.0>]
    message_queue_len: 1
    messages: [{'EXIT',<0.272.0>,normal}]
    links: [<0.270.0>,<0.44.0>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 610
    stack_size: 29
    reductions: 190
  neighbours:
2021-07-22 11:11:28 =CRASH REPORT====
  crasher:
    initial call: application_master:init/4
    pid: <0.265.0>
    registered_name: []
    exception exit: {{bad_return,{{'Elixir.Processor.Application',start,[normal,[]]},{'EXIT',{undef,[{'Elixir.Supervisor',start_link,[[{strategy,one_for_one},{name,'Elixir.Processor.Supervisor'}]],[]},{'Elixir.Processor.Application',start,2,[{file,"lib/processor/application.ex"},{line,10}]},{application_master,start_it_old,4,[{file,"application_master.erl"},{line,293}]}]}}}},[{application_master,init,4,[{file,"application_master.erl"},{line,142}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,226}]}]}
    ancestors: [<0.264.0>]
    message_queue_len: 1
    messages: [{'EXIT',<0.266.0>,normal}]
    links: [<0.264.0>,<0.44.0>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 610
    stack_size: 29
    reductions: 192
  neighbours:
2021-07-22 11:11:52 =CRASH REPORT====
  crasher:
    initial call: application_master:init/4
    pid: <0.265.0>
    registered_name: []
    exception exit: {{bad_return,{{'Elixir.Processor.Application',start,[normal,[]]},{'EXIT',{undef,[{'Elixir.Supervisor',start_link,[[{strategy,one_for_one},{name,'Elixir.Processor.Supervisor'}]],[]},{application_master,start_it_old,4,[{file,"application_master.erl"},{line,293}]}]}}}},[{application_master,init,4,[{file,"application_master.erl"},{line,142}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,226}]}]}
    ancestors: [<0.264.0>]
    message_queue_len: 1
    messages: [{'EXIT',<0.266.0>,normal}]
    links: [<0.264.0>,<0.44.0>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 610
    stack_size: 29
    reductions: 201
  neighbours:
2021-07-22 12:05:05 =SUPERVISOR REPORT====
     Supervisor: {local,'Elixir.Processor.Supervisor'}
     Context:    start_error
     Reason:     <0.280.0>
     Offender:   [{pid,undefined},{id,github},{mfargs,{'Elixir.Processor',start_link,[<<"github">>]}},{restart_type,permanent},{significant,false},{shutdown,500},{child_type,worker}]

2021-07-22 12:05:05 =CRASH REPORT====
  crasher:
    initial call: application_master:init/4
    pid: <0.277.0>
    registered_name: []
    exception exit: {{bad_return,{{'Elixir.Processor.Application',start,[normal,[]]},ok}},[{application_master,init,4,[{file,"application_master.erl"},{line,142}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,226}]}]}
    ancestors: [<0.276.0>]
    message_queue_len: 1
    messages: [{'EXIT',<0.278.0>,normal}]
    links: [<0.276.0>,<0.44.0>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 376
    stack_size: 29
    reductions: 208
  neighbours:
2021-07-22 12:05:15 =SUPERVISOR REPORT====
     Supervisor: {local,'Elixir.Processor.Supervisor'}
     Context:    start_error
     Reason:     <0.254.0>
     Offender:   [{pid,undefined},{id,github},{mfargs,{'Elixir.Processor',start_link,[<<"github">>]}},{restart_type,permanent},{significant,false},{shutdown,500},{child_type,worker}]

2021-07-22 12:05:15 =CRASH REPORT====
  crasher:
    initial call: application_master:init/4
    pid: <0.251.0>
    registered_name: []
    exception exit: {{bad_return,{{'Elixir.Processor.Application',start,[normal,[]]},ok}},[{application_master,init,4,[{file,"application_master.erl"},{line,142}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,226}]}]}
    ancestors: [<0.250.0>]
    message_queue_len: 1
    messages: [{'EXIT',<0.252.0>,normal}]
    links: [<0.250.0>,<0.44.0>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 376
    stack_size: 29
    reductions: 188
  neighbours:
2021-07-22 12:05:29 =SUPERVISOR REPORT====
     Supervisor: {local,'Elixir.Processor.Supervisor'}
     Context:    start_error
     Reason:     <0.254.0>
     Offender:   [{pid,undefined},{id,github},{mfargs,{'Elixir.Processor',start_link,[<<"github">>]}},{restart_type,permanent},{significant,false},{shutdown,500},{child_type,worker}]

2021-07-22 12:05:29 =CRASH REPORT====
  crasher:
    initial call: application_master:init/4
    pid: <0.251.0>
    registered_name: []
    exception exit: {{bad_return,{{'Elixir.Processor.Application',start,[normal,[]]},ok}},[{application_master,init,4,[{file,"application_master.erl"},{line,142}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,226}]}]}
    ancestors: [<0.250.0>]
    message_queue_len: 1
    messages: [{'EXIT',<0.252.0>,normal}]
    links: [<0.250.0>,<0.44.0>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 376
    stack_size: 29
    reductions: 188
  neighbours:
2021-07-22 12:05:42 =SUPERVISOR REPORT====
     Supervisor: {local,'Elixir.Processor.Supervisor'}
     Context:    start_error
     Reason:     ok
     Offender:   [{pid,undefined},{id,github},{mfargs,{'Elixir.Processor',start_link,[<<"github">>]}},{restart_type,permanent},{significant,false},{shutdown,500},{child_type,worker}]

2021-07-22 12:05:42 =CRASH REPORT====
  crasher:
    initial call: application_master:init/4
    pid: <0.266.0>
    registered_name: []
    exception exit: {{bad_return,{{'Elixir.Processor.Application',start,[normal,[]]},ok}},[{application_master,init,4,[{file,"application_master.erl"},{line,142}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,226}]}]}
    ancestors: [<0.265.0>]
    message_queue_len: 1
    messages: [{'EXIT',<0.267.0>,normal}]
    links: [<0.265.0>,<0.44.0>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 376
    stack_size: 29
    reductions: 198
  neighbours:
2021-07-22 12:06:19 =SUPERVISOR REPORT====
     Supervisor: {local,'Elixir.Processor.Supervisor'}
     Context:    start_error
     Reason:     ok
     Offender:   [{pid,undefined},{id,github},{mfargs,{'Elixir.Processor',start_link,[<<"github">>]}},{restart_type,permanent},{significant,false},{shutdown,500},{child_type,worker}]

2021-07-22 12:06:19 =CRASH REPORT====
  crasher:
    initial call: application_master:init/4
    pid: <0.266.0>
    registered_name: []
    exception exit: {{bad_return,{{'Elixir.Processor.Application',start,[normal,[]]},ok}},[{application_master,init,4,[{file,"application_master.erl"},{line,142}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,226}]}]}
    ancestors: [<0.265.0>]
    message_queue_len: 1
    messages: [{'EXIT',<0.267.0>,normal}]
    links: [<0.265.0>,<0.44.0>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 376
    stack_size: 29
    reductions: 188
  neighbours:
2021-07-22 12:07:59 =CRASH REPORT====
  crasher:
    initial call: application_master:init/4
    pid: <0.282.0>
    registered_name: []
    exception exit: {{bad_return,{{'Elixir.Processor.Application',start,[normal,[]]},{'EXIT',{undef,[{'Elixir.IO',pust,[<<"children = [{Processor, \"github\"}, {Processor, \"trello\"}]">>],[]},{'Elixir.Processor.Application',start,2,[{file,"lib/processor/application.ex"},{line,10}]},{application_master,start_it_old,4,[{file,"application_master.erl"},{line,293}]}]}}}},[{application_master,init,4,[{file,"application_master.erl"},{line,142}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,226}]}]}
    ancestors: [<0.281.0>]
    message_queue_len: 1
    messages: [{'EXIT',<0.283.0>,normal}]
    links: [<0.281.0>,<0.44.0>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 610
    stack_size: 29
    reductions: 190
  neighbours:
2021-07-22 12:08:27 =CRASH REPORT====
  crasher:
    initial call: application_master:init/4
    pid: <0.276.0>
    registered_name: []
    exception exit: {{bad_return,{{'Elixir.Processor.Application',start,[normal,[]]},{'EXIT',{undef,[{'Elixir.IO',pust,[<<"children = [{Processor, \"github\"}, {Processor, \"trello\"}]">>],[]},{'Elixir.Processor.Application',start,2,[{file,"lib/processor/application.ex"},{line,10}]},{application_master,start_it_old,4,[{file,"application_master.erl"},{line,293}]}]}}}},[{application_master,init,4,[{file,"application_master.erl"},{line,142}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,226}]}]}
    ancestors: [<0.275.0>]
    message_queue_len: 1
    messages: [{'EXIT',<0.277.0>,normal}]
    links: [<0.275.0>,<0.44.0>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 610
    stack_size: 29
    reductions: 190
  neighbours:
2021-07-22 12:16:55 =SUPERVISOR REPORT====
     Supervisor: {local,'Elixir.Processor.Supervisor'}
     Context:    start_error
     Reason:     {'EXIT',{noproc,{'Elixir.GenServer',call,['Elixir.AMQP.Channel.ReceiverManager',{register_handler,<0.268.0>,<0.197.0>,consume,[]},5000]}}}
     Offender:   [{pid,undefined},{id,github},{mfargs,{'Elixir.Processor',start_link,[<<"github">>]}},{restart_type,permanent},{significant,false},{shutdown,500},{child_type,worker}]

2021-07-22 12:22:08 =CRASH REPORT====
  crasher:
    initial call: application_master:init/4
    pid: <0.276.0>
    registered_name: []
    exception exit: {{bad_return,{{'Elixir.Processor.Application',start,[normal,[]]},{'EXIT',{undef,[{'Elixir.IO',pust,[<<"children = [{Processor, \"github\"}, {Processor, \"trello\"}]">>],[]},{'Elixir.Processor.Application',start,2,[{file,"lib/processor/application.ex"},{line,10}]},{application_master,start_it_old,4,[{file,"application_master.erl"},{line,293}]}]}}}},[{application_master,init,4,[{file,"application_master.erl"},{line,142}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,226}]}]}
    ancestors: [<0.275.0>]
    message_queue_len: 1
    messages: [{'EXIT',<0.277.0>,normal}]
    links: [<0.275.0>,<0.44.0>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 610
    stack_size: 29
    reductions: 194
  neighbours:
2021-07-22 12:36:09 =SUPERVISOR REPORT====
     Supervisor: {local,'Elixir.Processor.Supervisor'}
     Context:    start_error
     Reason:     <0.280.0>
     Offender:   [{pid,undefined},{id,github},{mfargs,{'Elixir.Processor',start_link,[<<"github">>]}},{restart_type,permanent},{significant,false},{shutdown,500},{child_type,worker}]

2021-07-22 12:36:09 =CRASH REPORT====
  crasher:
    initial call: application_master:init/4
    pid: <0.277.0>
    registered_name: []
    exception exit: {{bad_return,{{'Elixir.Processor.Application',start,[normal,[]]},ok}},[{application_master,init,4,[{file,"application_master.erl"},{line,142}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,226}]}]}
    ancestors: [<0.276.0>]
    message_queue_len: 1
    messages: [{'EXIT',<0.278.0>,normal}]
    links: [<0.276.0>,<0.44.0>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 376
    stack_size: 29
    reductions: 198
  neighbours:
2021-07-22 12:36:37 =CRASH REPORT====
  crasher:
    initial call: application_master:init/4
    pid: <0.277.0>
    registered_name: []
    exception exit: {{bad_return,{{'Elixir.Processor.Application',start,[normal,[]]},ok}},[{application_master,init,4,[{file,"application_master.erl"},{line,142}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,226}]}]}
    ancestors: [<0.276.0>]
    message_queue_len: 1
    messages: [{'EXIT',<0.278.0>,normal}]
    links: [<0.276.0>,<0.44.0>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 376
    stack_size: 29
    reductions: 198
  neighbours:
2021-07-22 12:40:29 =CRASH REPORT====
  crasher:
    pid: <0.279.0>
    registered_name: []
    exception error: bad argument: [{erlang,apply,[{'Elixir.Processor',<<"trello">>},init,[[{strategy,one_for_one},{name,'Elixir.Processor.Supervisor'}]]],[{error_info,#{module => erl_erts_errors}}]},{supervisor,init,1,[{file,"supervisor.erl"},{line,330}]},{gen_server,init_it,2,[{file,"gen_server.erl"},{line,423}]},{gen_server,init_it,6,[{file,"gen_server.erl"},{line,390}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,226}]}]
    ancestors: [<0.277.0>]
    message_queue_len: 0
    messages: []
    links: [<0.277.0>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 376
    stack_size: 27
    reductions: 188
  neighbours:
2021-07-22 12:40:29 =CRASH REPORT====
  crasher:
    pid: <0.278.0>
    registered_name: []
    exception error: bad argument: [{erlang,apply,[{'Elixir.Processor',<<"github">>},init,[[{strategy,one_for_one},{name,'Elixir.Processor.Supervisor'}]]],[{error_info,#{module => erl_erts_errors}}]},{supervisor,init,1,[{file,"supervisor.erl"},{line,330}]},{gen_server,init_it,2,[{file,"gen_server.erl"},{line,423}]},{gen_server,init_it,6,[{file,"gen_server.erl"},{line,390}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,226}]}]
    ancestors: [<0.277.0>]
    message_queue_len: 0
    messages: []
    links: [<0.277.0>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 610
    stack_size: 27
    reductions: 193
  neighbours:
2021-07-22 12:40:29 =CRASH REPORT====
  crasher:
    initial call: application_master:init/4
    pid: <0.276.0>
    registered_name: []
    exception exit: {{{badarg,[{erlang,apply,[{'Elixir.Processor',<<"github">>},init,[[{strategy,one_for_one},{name,'Elixir.Processor.Supervisor'}]]],[{error_info,#{module => erl_erts_errors}}]},{supervisor,init,1,[{file,"supervisor.erl"},{line,330}]},{gen_server,init_it,2,[{file,"gen_server.erl"},{line,423}]},{gen_server,init_it,6,[{file,"gen_server.erl"},{line,390}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,226}]}]},{'Elixir.Processor.Application',start,[normal,[]]}},[{application_master,init,4,[{file,"application_master.erl"},{line,142}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,226}]}]}
    ancestors: [<0.275.0>]
    message_queue_len: 1
    messages: [{'EXIT',<0.277.0>,normal}]
    links: [<0.275.0>,<0.44.0>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 610
    stack_size: 29
    reductions: 211
  neighbours:
2021-07-22 14:35:02 =CRASH REPORT====
  crasher:
    initial call: application_master:init/4
    pid: <0.271.0>
    registered_name: []
    exception exit: {{bad_return,{{'Elixir.Processor.Application',start,[normal,[]]},{'EXIT',{#{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"The module Processor was given as a child to a supervisor\nbut it does not implement child_spec/1.\n\nIf you own the given module, please define a child_spec/1 function\nthat receives an argument and returns a child specification as a map.\nFor example:\n\n    def child_spec(opts) do\n      %{\n        id: __MODULE__,\n        start: {__MODULE__, :start_link, [opts]},\n        type: :worker,\n        restart: :permanent,\n        shutdown: 500\n      }\n    end\n\nNote that \"use Agent\", \"use GenServer\" and so on automatically define\nthis function for you.\n\nHowever, if you don't own the given module and it doesn't implement\nchild_spec/1, instead of passing the module name directly as a supervisor\nchild, you will have to pass a child specification as a map:\n\n    %{\n      id: Processor,\n      start: {Processor, :start_link, [arg1, arg2]}\n    }\n\nSee the Supervisor documentation for more information.\n">>},[{'Elixir.Supervisor',init_child,1,[{file,"lib/supervisor.ex"},{line,631}]},{'Elixir.Enum','-map/2-lists^map/1-0-',2,[{file,"lib/enum.ex"},{line,1553}]},{'Elixir.Supervisor',init,2,[{file,"lib/supervisor.ex"},{line,617}]},{'Elixir.Supervisor',start_link,2,[{file,"lib/supervisor.ex"},{line,556}]},{application_master,start_it_old,4,[{file,"application_master.erl"},{line,293}]}]}}}},[{application_master,init,4,[{file,"application_master.erl"},{line,142}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,226}]}]}
    ancestors: [<0.270.0>]
    message_queue_len: 1
    messages: [{'EXIT',<0.272.0>,normal}]
    links: [<0.270.0>,<0.44.0>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 610
    stack_size: 29
    reductions: 211
  neighbours:
2021-07-22 14:35:18 =CRASH REPORT====
  crasher:
    initial call: supervisor:Elixir.Supervisor.Default/1
    pid: <0.268.0>
    registered_name: []
    exception exit: {{start_spec,{duplicate_child_name,'Elixir.Processor'}},[{gen_server,init_it,6,[{file,"gen_server.erl"},{line,407}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,226}]}]}
    ancestors: [<0.267.0>]
    message_queue_len: 1
    messages: [{'EXIT',<0.267.0>,normal}]
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 376
    stack_size: 29
    reductions: 244
  neighbours:
2021-07-22 14:35:18 =CRASH REPORT====
  crasher:
    initial call: application_master:init/4
    pid: <0.266.0>
    registered_name: []
    exception exit: {{{start_spec,{duplicate_child_name,'Elixir.Processor'}},{'Elixir.Processor.Application',start,[normal,[]]}},[{application_master,init,4,[{file,"application_master.erl"},{line,142}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,226}]}]}
    ancestors: [<0.265.0>]
    message_queue_len: 1
    messages: [{'EXIT',<0.267.0>,normal}]
    links: [<0.265.0>,<0.44.0>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 376
    stack_size: 29
    reductions: 202
  neighbours:
2021-07-22 14:35:41 =SUPERVISOR REPORT====
     Supervisor: {local,'Elixir.Processor.Supervisor'}
     Context:    start_error
     Reason:     {already_started,<0.269.0>}
     Offender:   [{pid,undefined},{id,trello},{mfargs,{'Elixir.Processor',start_link,[<<"trello">>]}},{restart_type,permanent},{significant,false},{shutdown,500},{child_type,worker}]

2021-07-22 14:35:41 =CRASH REPORT====
  crasher:
    initial call: application_master:init/4
    pid: <0.266.0>
    registered_name: []
    exception exit: {{{shutdown,{failed_to_start_child,trello,{already_started,<0.269.0>}}},{'Elixir.Processor.Application',start,[normal,[]]}},[{application_master,init,4,[{file,"application_master.erl"},{line,142}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,226}]}]}
    ancestors: [<0.265.0>]
    message_queue_len: 1
    messages: [{'EXIT',<0.267.0>,normal}]
    links: [<0.265.0>,<0.44.0>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 610
    stack_size: 29
    reductions: 198
  neighbours:
2021-07-22 14:36:16 =SUPERVISOR REPORT====
     Supervisor: {local,'Elixir.Processor.Supervisor'}
     Context:    start_error
     Reason:     {already_started,<0.269.0>}
     Offender:   [{pid,undefined},{id,trello},{mfargs,{'Elixir.Processor',start_link,[<<"trello">>]}},{restart_type,permanent},{significant,false},{shutdown,500},{child_type,worker}]

2021-07-22 14:37:02 =SUPERVISOR REPORT====
     Supervisor: {local,'Elixir.Processor.Supervisor'}
     Context:    start_error
     Reason:     {already_started,<0.268.0>}
     Offender:   [{pid,undefined},{id,trello},{mfargs,{'Elixir.Processor',start_link,[<<"trello">>]}},{restart_type,permanent},{significant,false},{shutdown,500},{child_type,worker}]

2021-07-22 14:37:02 =CRASH REPORT====
  crasher:
    initial call: application_master:init/4
    pid: <0.265.0>
    registered_name: []
    exception exit: {{{shutdown,{failed_to_start_child,trello,{already_started,<0.268.0>}}},{'Elixir.Processor.Application',start,[normal,[]]}},[{application_master,init,4,[{file,"application_master.erl"},{line,142}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,226}]}]}
    ancestors: [<0.264.0>]
    message_queue_len: 1
    messages: [{'EXIT',<0.266.0>,normal}]
    links: [<0.264.0>,<0.44.0>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 610
    stack_size: 29
    reductions: 208
  neighbours:
2021-07-22 14:37:23 =SUPERVISOR REPORT====
     Supervisor: {local,'Elixir.Processor.Supervisor'}
     Context:    start_error
     Reason:     {already_started,<0.254.0>}
     Offender:   [{pid,undefined},{id,trello},{mfargs,{'Elixir.Processor',start_link,[<<"trello">>]}},{restart_type,permanent},{significant,false},{shutdown,500},{child_type,worker}]

2021-07-22 14:37:23 =CRASH REPORT====
  crasher:
    initial call: application_master:init/4
    pid: <0.251.0>
    registered_name: []
    exception exit: {{{shutdown,{failed_to_start_child,trello,{already_started,<0.254.0>}}},{'Elixir.Processor.Application',start,[normal,[]]}},[{application_master,init,4,[{file,"application_master.erl"},{line,142}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,226}]}]}
    ancestors: [<0.250.0>]
    message_queue_len: 1
    messages: [{'EXIT',<0.252.0>,normal}]
    links: [<0.250.0>,<0.44.0>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 610
    stack_size: 29
    reductions: 208
  neighbours:
2021-07-22 14:37:33 =SUPERVISOR REPORT====
     Supervisor: {local,'Elixir.Processor.Supervisor'}
     Context:    start_error
     Reason:     {already_started,<0.265.0>}
     Offender:   [{pid,undefined},{id,trello},{mfargs,{'Elixir.Processor',start_link,[<<"trello">>]}},{restart_type,permanent},{significant,false},{shutdown,500},{child_type,worker}]

2021-07-22 14:37:33 =CRASH REPORT====
  crasher:
    initial call: application_master:init/4
    pid: <0.262.0>
    registered_name: []
    exception exit: {{{shutdown,{failed_to_start_child,trello,{already_started,<0.265.0>}}},{'Elixir.Processor.Application',start,[normal,[]]}},[{application_master,init,4,[{file,"application_master.erl"},{line,142}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,226}]}]}
    ancestors: [<0.261.0>]
    message_queue_len: 1
    messages: [{'EXIT',<0.263.0>,normal}]
    links: [<0.261.0>,<0.44.0>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 610
    stack_size: 29
    reductions: 208
  neighbours:
2021-07-22 14:39:40 =CRASH REPORT====
  crasher:
    initial call: application_master:init/4
    pid: <0.276.0>
    registered_name: []
    exception exit: {{bad_return,{{'Elixir.Processor.Application',start,[normal,[]]},{'EXIT',{badarg,[{erlang,binary_to_atom,[[<<"github">>],utf8],[{error_info,#{module => erl_erts_errors}}]},{'Elixir.Processor',child_spec,1,[{file,"lib/processor.ex"},{line,36}]},{'Elixir.Supervisor',init_child,1,[{file,"lib/supervisor.ex"},{line,626}]},{'Elixir.Enum','-map/2-lists^map/1-0-',2,[{file,"lib/enum.ex"},{line,1553}]},{'Elixir.Supervisor',init,2,[{file,"lib/supervisor.ex"},{line,617}]},{'Elixir.Supervisor',start_link,2,[{file,"lib/supervisor.ex"},{line,556}]},{application_master,start_it_old,4,[{file,"application_master.erl"},{line,293}]}]}}}},[{application_master,init,4,[{file,"application_master.erl"},{line,142}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,226}]}]}
    ancestors: [<0.275.0>]
    message_queue_len: 1
    messages: [{'EXIT',<0.277.0>,normal}]
    links: [<0.275.0>,<0.44.0>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 610
    stack_size: 29
    reductions: 216
  neighbours:
2021-07-22 14:40:12 =SUPERVISOR REPORT====
     Supervisor: {local,'Elixir.Processor.Supervisor'}
     Context:    start_error
     Reason:     {badarg,[{erlang,byte_size,[[<<"github">>]],[{error_info,#{module => erl_erts_errors}}]},{'Elixir.Processor',start_connection,1,[{file,"lib/processor.ex"},{line,50}]},{'Elixir.Processor',init,1,[{file,"lib/processor.ex"},{line,18}]},{gen_server,init_it,2,[{file,"gen_server.erl"},{line,423}]},{gen_server,init_it,6,[{file,"gen_server.erl"},{line,390}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,226}]}]}
     Offender:   [{pid,undefined},{id,'Elixir.Processor'},{mfargs,{'Elixir.Processor',start_link,[[<<"github">>]]}},{restart_type,permanent},{significant,false},{shutdown,5000},{child_type,worker}]

2021-07-22 14:40:12 =CRASH REPORT====
  crasher:
    initial call: application_master:init/4
    pid: <0.277.0>
    registered_name: []
    exception exit: {{{shutdown,{failed_to_start_child,'Elixir.Processor',{badarg,[{erlang,byte_size,[[<<"github">>]],[{error_info,#{module => erl_erts_errors}}]},{'Elixir.Processor',start_connection,1,[{file,"lib/processor.ex"},{line,50}]},{'Elixir.Processor',init,1,[{file,"lib/processor.ex"},{line,18}]},{gen_server,init_it,2,[{file,"gen_server.erl"},{line,423}]},{gen_server,init_it,6,[{file,"gen_server.erl"},{line,390}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,226}]}]}}},{'Elixir.Processor.Application',start,[normal,[]]}},[{application_master,init,4,[{file,"application_master.erl"},{line,142}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,226}]}]}
    ancestors: [<0.276.0>]
    message_queue_len: 1
    messages: [{'EXIT',<0.278.0>,normal}]
    links: [<0.276.0>,<0.44.0>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 610
    stack_size: 29
    reductions: 200
  neighbours:
2021-07-22 14:41:26 =CRASH REPORT====
  crasher:
    initial call: Elixir.Processor:init/1
    pid: <0.279.0>
    registered_name: []
    exception error: bad argument: [{erlang,byte_size,[["github"]],[{error_info,#{module => erl_erts_errors}}]},{'Elixir.Processor',start_connection,1,[{file,"lib/processor.ex"},{line,50}]},{'Elixir.Processor',init,1,[{file,"lib/processor.ex"},{line,18}]},{gen_server,init_it,2,[{file,"gen_server.erl"},{line,423}]},{gen_server,init_it,6,[{file,"gen_server.erl"},{line,390}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,226}]}]
    ancestors: ['Elixir.Processor.Supervisor',<0.277.0>]
    message_queue_len: 0
    messages: []
    links: [<0.278.0>]
    dictionary: [{gen_server_call_timeout,60000}]
    trap_exit: false
    status: running
    heap_size: 2586
    stack_size: 29
    reductions: 7316
  neighbours:
2021-07-22 14:41:26 =SUPERVISOR REPORT====
     Supervisor: {local,'Elixir.Processor.Supervisor'}
     Context:    start_error
     Reason:     {badarg,[{erlang,byte_size,[["github"]],[{error_info,#{module => erl_erts_errors}}]},{'Elixir.Processor',start_connection,1,[{file,"lib/processor.ex"},{line,50}]},{'Elixir.Processor',init,1,[{file,"lib/processor.ex"},{line,18}]},{gen_server,init_it,2,[{file,"gen_server.erl"},{line,423}]},{gen_server,init_it,6,[{file,"gen_server.erl"},{line,390}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,226}]}]}
     Offender:   [{pid,undefined},{id,'Elixir.Processor'},{mfargs,{'Elixir.Processor',start_link,[["github"]]}},{restart_type,permanent},{significant,false},{shutdown,5000},{child_type,worker}]

2021-07-22 14:41:26 =CRASH REPORT====
  crasher:
    initial call: application_master:init/4
    pid: <0.276.0>
    registered_name: []
    exception exit: {{{shutdown,{failed_to_start_child,'Elixir.Processor',{badarg,[{erlang,byte_size,[["github"]],[{error_info,#{module => erl_erts_errors}}]},{'Elixir.Processor',start_connection,1,[{file,"lib/processor.ex"},{line,50}]},{'Elixir.Processor',init,1,[{file,"lib/processor.ex"},{line,18}]},{gen_server,init_it,2,[{file,"gen_server.erl"},{line,423}]},{gen_server,init_it,6,[{file,"gen_server.erl"},{line,390}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,226}]}]}}},{'Elixir.Processor.Application',start,[normal,[]]}},[{application_master,init,4,[{file,"application_master.erl"},{line,142}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,226}]}]}
    ancestors: [<0.275.0>]
    message_queue_len: 1
    messages: [{'EXIT',<0.277.0>,normal}]
    links: [<0.275.0>,<0.44.0>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 610
    stack_size: 29
    reductions: 200
  neighbours:
2021-07-22 14:44:21 =ERROR REPORT====
** Generic server 'Elixir.Processor' terminating 
** Last message in was {basic_consume_ok,#{consumer_tag => <<"amq.ctag-FwFrMD1SrbUqxOkhEQWltA">>}}
** When Server state == #{channel => #{'__struct__' => 'Elixir.AMQP.Channel',conn => #{'__struct__' => 'Elixir.AMQP.Connection',pid => <0.283.0>},custom_consumer => nil,pid => <0.294.0>},source => <<"github">>}
** Reason for termination ==
** {bad_return_value,ok}
2021-07-22 14:44:21 =CRASH REPORT====
  crasher:
    initial call: Elixir.Processor:init/1
    pid: <0.280.0>
    registered_name: 'Elixir.Processor'
    exception exit: {{bad_return_value,ok},[{gen_server,handle_common_reply,8,[{file,"gen_server.erl"},{line,815}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,226}]}]}
    ancestors: ['Elixir.Processor.Supervisor',<0.278.0>]
    message_queue_len: 0
    messages: []
    links: [<0.279.0>]
    dictionary: [{gen_server_call_timeout,60000}]
    trap_exit: false
    status: running
    heap_size: 2586
    stack_size: 29
    reductions: 9903
  neighbours:
2021-07-22 14:44:21 =SUPERVISOR REPORT====
     Supervisor: {local,'Elixir.Processor.Supervisor'}
     Context:    child_terminated
     Reason:     {bad_return_value,ok}
     Offender:   [{pid,<0.280.0>},{id,'Elixir.Processor'},{mfargs,{'Elixir.Processor',start_link,[<<"github">>]}},{restart_type,permanent},{significant,false},{shutdown,5000},{child_type,worker}]

2021-07-22 14:44:21 =SUPERVISOR REPORT====
     Supervisor: {local,'Elixir.Processor.Supervisor'}
     Context:    start_error
     Reason:     {{shutdown,{server_initiated_close,405,<<"RESOURCE_LOCKED - cannot obtain exclusive access to locked queue 'raw_input' in vhost '/'. It could be originally declared on another connection or the exclusive property value does not match that of the original declaration.">>}},{gen_server,call,[<0.311.0>,{call,{'queue.declare',0,<<"raw_input">>,false,false,true,false,false,[]},none,<0.298.0>},60000]}}
     Offender:   [{pid,<0.280.0>},{id,'Elixir.Processor'},{mfargs,{'Elixir.Processor',start_link,[<<"github">>]}},{restart_type,permanent},{significant,false},{shutdown,5000},{child_type,worker}]

2021-07-22 14:44:21 =CRASH REPORT====
  crasher:
    initial call: Elixir.Processor:init/1
    pid: <0.298.0>
    registered_name: []
    exception exit: {{{shutdown,{server_initiated_close,405,<<"RESOURCE_LOCKED - cannot obtain exclusive access to locked queue 'raw_input' in vhost '/'. It could be originally declared on another connection or the exclusive property value does not match that of the original declaration.">>}},{gen_server,call,[<0.311.0>,{call,{'queue.declare',0,<<"raw_input">>,false,false,true,false,false,[]},none,<0.298.0>},60000]}},[{gen_server,call,3,[{file,"gen_server.erl"},{line,247}]},{'Elixir.AMQP.Queue',declare,3,[{file,"lib/amqp/queue.ex"},{line,45}]},{'Elixir.Processor',create_queue,3,[{file,"lib/processor.ex"},{line,65}]},{'Elixir.Processor',start_connection,1,[{file,"lib/processor.ex"},{line,50}]},{'Elixir.Processor',init,1,[{file,"lib/processor.ex"},{line,18}]},{gen_server,init_it,2,[{file,"gen_server.erl"},{line,423}]},{gen_server,init_it,6,[{file,"gen_server.erl"},{line,390}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,226}]}]}
    ancestors: ['Elixir.Processor.Supervisor',<0.278.0>]
    message_queue_len: 0
    messages: []
    links: [<0.279.0>]
    dictionary: [{gen_server_call_timeout,60000}]
    trap_exit: false
    status: running
    heap_size: 2586
    stack_size: 29
    reductions: 7189
  neighbours:
2021-07-22 14:44:21 =SUPERVISOR REPORT====
     Supervisor: {local,'Elixir.Processor.Supervisor'}
     Context:    start_error
     Reason:     {{shutdown,{server_initiated_close,405,<<"RESOURCE_LOCKED - cannot obtain exclusive access to locked queue 'raw_input' in vhost '/'. It could be originally declared on another connection or the exclusive property value does not match that of the original declaration.">>}},{gen_server,call,[<0.325.0>,{call,{'queue.declare',0,<<"raw_input">>,false,false,true,false,false,[]},none,<0.313.0>},60000]}}
     Offender:   [{pid,{restarting,<0.280.0>}},{id,'Elixir.Processor'},{mfargs,{'Elixir.Processor',start_link,[<<"github">>]}},{restart_type,permanent},{significant,false},{shutdown,5000},{child_type,worker}]

2021-07-22 14:44:21 =CRASH REPORT====
  crasher:
    initial call: Elixir.Processor:init/1
    pid: <0.313.0>
    registered_name: []
    exception exit: {{{shutdown,{server_initiated_close,405,<<"RESOURCE_LOCKED - cannot obtain exclusive access to locked queue 'raw_input' in vhost '/'. It could be originally declared on another connection or the exclusive property value does not match that of the original declaration.">>}},{gen_server,call,[<0.325.0>,{call,{'queue.declare',0,<<"raw_input">>,false,false,true,false,false,[]},none,<0.313.0>},60000]}},[{gen_server,call,3,[{file,"gen_server.erl"},{line,247}]},{'Elixir.AMQP.Queue',declare,3,[{file,"lib/amqp/queue.ex"},{line,45}]},{'Elixir.Processor',create_queue,3,[{file,"lib/processor.ex"},{line,65}]},{'Elixir.Processor',start_connection,1,[{file,"lib/processor.ex"},{line,50}]},{'Elixir.Processor',init,1,[{file,"lib/processor.ex"},{line,18}]},{gen_server,init_it,2,[{file,"gen_server.erl"},{line,423}]},{gen_server,init_it,6,[{file,"gen_server.erl"},{line,390}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,226}]}]}
    ancestors: ['Elixir.Processor.Supervisor',<0.278.0>]
    message_queue_len: 0
    messages: []
    links: [<0.279.0>]
    dictionary: [{gen_server_call_timeout,60000}]
    trap_exit: false
    status: running
    heap_size: 2586
    stack_size: 29
    reductions: 7189
  neighbours:
2021-07-22 14:44:21 =SUPERVISOR REPORT====
     Supervisor: {local,'Elixir.Processor.Supervisor'}
     Context:    start_error
     Reason:     {{shutdown,{server_initiated_close,405,<<"RESOURCE_LOCKED - cannot obtain exclusive access to locked queue 'raw_input' in vhost '/'. It could be originally declared on another connection or the exclusive property value does not match that of the original declaration.">>}},{gen_server,call,[<0.339.0>,{call,{'queue.declare',0,<<"raw_input">>,false,false,true,false,false,[]},none,<0.327.0>},60000]}}
     Offender:   [{pid,{restarting,<0.280.0>}},{id,'Elixir.Processor'},{mfargs,{'Elixir.Processor',start_link,[<<"github">>]}},{restart_type,permanent},{significant,false},{shutdown,5000},{child_type,worker}]

2021-07-22 14:44:21 =SUPERVISOR REPORT====
     Supervisor: {local,'Elixir.Processor.Supervisor'}
     Context:    shutdown
     Reason:     reached_max_restart_intensity
     Offender:   [{pid,{restarting,<0.280.0>}},{id,'Elixir.Processor'},{mfargs,{'Elixir.Processor',start_link,[<<"github">>]}},{restart_type,permanent},{significant,false},{shutdown,5000},{child_type,worker}]

2021-07-22 14:47:16 =CRASH REPORT====
  crasher:
    initial call: supervisor:Elixir.Supervisor.Default/1
    pid: <0.278.0>
    registered_name: []
    exception exit: {{start_spec,{duplicate_child_name,'Elixir.Processor'}},[{gen_server,init_it,6,[{file,"gen_server.erl"},{line,407}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,226}]}]}
    ancestors: [<0.277.0>]
    message_queue_len: 1
    messages: [{'EXIT',<0.277.0>,normal}]
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 376
    stack_size: 29
    reductions: 240
  neighbours:
2021-07-22 14:47:16 =CRASH REPORT====
  crasher:
    initial call: application_master:init/4
    pid: <0.276.0>
    registered_name: []
    exception exit: {{{start_spec,{duplicate_child_name,'Elixir.Processor'}},{'Elixir.Processor.Application',start,[normal,[]]}},[{application_master,init,4,[{file,"application_master.erl"},{line,142}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,226}]}]}
    ancestors: [<0.275.0>]
    message_queue_len: 1
    messages: [{'EXIT',<0.277.0>,normal}]
    links: [<0.275.0>,<0.44.0>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 376
    stack_size: 29
    reductions: 190
  neighbours:
2021-07-22 14:47:33 =SUPERVISOR REPORT====
     Supervisor: {local,'Elixir.Processor.Supervisor'}
     Context:    start_error
     Reason:     {already_started,<0.280.0>}
     Offender:   [{pid,undefined},{id,trello},{mfargs,{'Elixir.Processor',start_link,[<<"trello">>]}},{restart_type,permanent},{significant,false},{shutdown,500},{child_type,worker}]

2021-07-22 14:47:33 =CRASH REPORT====
  crasher:
    initial call: application_master:init/4
    pid: <0.277.0>
    registered_name: []
    exception exit: {{{shutdown,{failed_to_start_child,trello,{already_started,<0.280.0>}}},{'Elixir.Processor.Application',start,[normal,[]]}},[{application_master,init,4,[{file,"application_master.erl"},{line,142}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,226}]}]}
    ancestors: [<0.276.0>]
    message_queue_len: 1
    messages: [{'EXIT',<0.278.0>,normal}]
    links: [<0.276.0>,<0.44.0>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 610
    stack_size: 29
    reductions: 208
  neighbours:
2021-07-22 14:50:32 =SUPERVISOR REPORT====
     Supervisor: {local,'Elixir.Processor.Supervisor'}
     Context:    start_error
     Reason:     {already_started,<0.280.0>}
     Offender:   [{pid,undefined},{id,trello},{mfargs,{'Elixir.Processor',start_link,[<<"trello">>]}},{restart_type,permanent},{significant,false},{shutdown,500},{child_type,worker}]

2021-07-22 14:50:32 =CRASH REPORT====
  crasher:
    initial call: application_master:init/4
    pid: <0.277.0>
    registered_name: []
    exception exit: {{{shutdown,{failed_to_start_child,trello,{already_started,<0.280.0>}}},{'Elixir.Processor.Application',start,[normal,[]]}},[{application_master,init,4,[{file,"application_master.erl"},{line,142}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,226}]}]}
    ancestors: [<0.276.0>]
    message_queue_len: 1
    messages: [{'EXIT',<0.278.0>,normal}]
    links: [<0.276.0>,<0.44.0>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 610
    stack_size: 29
    reductions: 208
  neighbours:
2021-07-22 14:51:24 =SUPERVISOR REPORT====
     Supervisor: {local,'Elixir.Processor.Supervisor'}
     Context:    start_error
     Reason:     {already_started,<0.280.0>}
     Offender:   [{pid,undefined},{id,trello},{mfargs,{'Elixir.Processor',start_link,[<<"trello">>]}},{restart_type,permanent},{significant,false},{shutdown,500},{child_type,worker}]

2021-07-22 14:51:24 =CRASH REPORT====
  crasher:
    initial call: application_master:init/4
    pid: <0.277.0>
    registered_name: []
    exception exit: {{{shutdown,{failed_to_start_child,trello,{already_started,<0.280.0>}}},{'Elixir.Processor.Application',start,[normal,[]]}},[{application_master,init,4,[{file,"application_master.erl"},{line,142}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,226}]}]}
    ancestors: [<0.276.0>]
    message_queue_len: 1
    messages: [{'EXIT',<0.278.0>,normal}]
    links: [<0.276.0>,<0.44.0>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 610
    stack_size: 29
    reductions: 208
  neighbours:
2021-07-22 15:00:29 =SUPERVISOR REPORT====
     Supervisor: {local,'Elixir.Processor.Supervisor'}
     Context:    start_error
     Reason:     {already_started,<0.279.0>}
     Offender:   [{pid,undefined},{id,'Elixir.Trello'},{mfargs,{'Elixir.Processor',start_link,[<<"trello">>]}},{restart_type,permanent},{significant,false},{shutdown,5000},{child_type,worker}]

2021-07-22 15:00:29 =CRASH REPORT====
  crasher:
    initial call: application_master:init/4
    pid: <0.276.0>
    registered_name: []
    exception exit: {{{shutdown,{failed_to_start_child,'Elixir.Trello',{already_started,<0.279.0>}}},{'Elixir.Processor.Application',start,[normal,[]]}},[{application_master,init,4,[{file,"application_master.erl"},{line,142}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,226}]}]}
    ancestors: [<0.275.0>]
    message_queue_len: 1
    messages: [{'EXIT',<0.277.0>,normal}]
    links: [<0.275.0>,<0.44.0>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 610
    stack_size: 29
    reductions: 208
  neighbours:
2021-07-22 15:05:20 =SUPERVISOR REPORT====
     Supervisor: {local,'Elixir.Processor.Supervisor'}
     Context:    start_error
     Reason:     {already_started,<0.279.0>}
     Offender:   [{pid,undefined},{id,'Elixir.Trello'},{mfargs,{'Elixir.Processor',start_link,[<<"trello">>]}},{restart_type,permanent},{significant,false},{shutdown,5000},{child_type,worker}]

2021-07-22 15:05:20 =CRASH REPORT====
  crasher:
    initial call: application_master:init/4
    pid: <0.276.0>
    registered_name: []
    exception exit: {{{shutdown,{failed_to_start_child,'Elixir.Trello',{already_started,<0.279.0>}}},{'Elixir.Processor.Application',start,[normal,[]]}},[{application_master,init,4,[{file,"application_master.erl"},{line,142}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,226}]}]}
    ancestors: [<0.275.0>]
    message_queue_len: 1
    messages: [{'EXIT',<0.277.0>,normal}]
    links: [<0.275.0>,<0.44.0>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 610
    stack_size: 29
    reductions: 208
  neighbours:
2021-07-22 15:08:23 =SUPERVISOR REPORT====
     Supervisor: {local,'Elixir.Processor.Supervisor'}
     Context:    start_error
     Reason:     {'EXIT',{#{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"expected :name option to be one of the following:\n\n  * nil\n  * atom\n  * {:global, term}\n  * {:via, module, term}\n\nGot: \"github\"\n">>},[{'Elixir.GenServer',do_start,4,[{file,"lib/gen_server.ex"},{line,944}]},{supervisor,do_start_child_i,3,[{file,"supervisor.erl"},{line,414}]},{supervisor,do_start_child,2,[{file,"supervisor.erl"},{line,400}]},{supervisor,'-start_children/2-fun-0-',3,[{file,"supervisor.erl"},{line,384}]},{supervisor,children_map,4,[{file,"supervisor.erl"},{line,1234}]},{supervisor,init_children,2,[{file,"supervisor.erl"},{line,350}]},{gen_server,init_it,2,[{file,"gen_server.erl"},{line,423}]},{gen_server,init_it,6,[{file,"gen_server.erl"},{line,390}]}]}}
     Offender:   [{pid,undefined},{id,'Elixir.Git'},{mfargs,{'Elixir.Processor',start_link,[<<"github">>]}},{restart_type,permanent},{significant,false},{shutdown,5000},{child_type,worker}]

2021-07-22 15:08:23 =CRASH REPORT====
  crasher:
    initial call: application_master:init/4
    pid: <0.282.0>
    registered_name: []
    exception exit: {{{shutdown,{failed_to_start_child,'Elixir.Git',{'EXIT',{#{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"expected :name option to be one of the following:\n\n  * nil\n  * atom\n  * {:global, term}\n  * {:via, module, term}\n\nGot: \"github\"\n">>},[{'Elixir.GenServer',do_start,4,[{file,"lib/gen_server.ex"},{line,944}]},{supervisor,do_start_child_i,3,[{file,"supervisor.erl"},{line,414}]},{supervisor,do_start_child,2,[{file,"supervisor.erl"},{line,400}]},{supervisor,'-start_children/2-fun-0-',3,[{file,"supervisor.erl"},{line,384}]},{supervisor,children_map,4,[{file,"supervisor.erl"},{line,1234}]},{supervisor,init_children,2,[{file,"supervisor.erl"},{line,350}]},{gen_server,init_it,2,[{file,"gen_server.erl"},{line,423}]},{gen_server,init_it,6,[{file,"gen_server.erl"},{line,390}]}]}}}},{'Elixir.Processor.Application',start,[normal,[]]}},[{application_master,init,4,[{file,"application_master.erl"},{line,142}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,226}]}]}
    ancestors: [<0.281.0>]
    message_queue_len: 1
    messages: [{'EXIT',<0.283.0>,normal}]
    links: [<0.281.0>,<0.44.0>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 987
    stack_size: 29
    reductions: 227
  neighbours:
2021-07-22 15:09:12 =SUPERVISOR REPORT====
     Supervisor: {local,'Elixir.Processor.Supervisor'}
     Context:    start_error
     Reason:     {{shutdown,{server_initiated_close,405,<<"RESOURCE_LOCKED - cannot obtain exclusive access to locked queue 'raw_input' in vhost '/'. It could be originally declared on another connection or the exclusive property value does not match that of the original declaration.">>}},{gen_server,call,[<0.309.0>,{call,{'queue.declare',0,<<"raw_input">>,false,false,true,false,false,[]},none,<0.297.0>},60000]}}
     Offender:   [{pid,undefined},{id,'Elixir.Trello'},{mfargs,{'Elixir.Processor',start_link,[<<"trello">>]}},{restart_type,permanent},{significant,false},{shutdown,5000},{child_type,worker}]

2021-07-22 15:09:12 =CRASH REPORT====
  crasher:
    initial call: application_master:init/4
    pid: <0.277.0>
    registered_name: []
    exception exit: {{{shutdown,{failed_to_start_child,'Elixir.Trello',{{shutdown,{server_initiated_close,405,<<"RESOURCE_LOCKED - cannot obtain exclusive access to locked queue 'raw_input' in vhost '/'. It could be originally declared on another connection or the exclusive property value does not match that of the original declaration.">>}},{gen_server,call,[<0.309.0>,{call,{'queue.declare',0,<<"raw_input">>,false,false,true,false,false,[]},none,<0.297.0>},60000]}}}},{'Elixir.Processor.Application',start,[normal,[]]}},[{application_master,init,4,[{file,"application_master.erl"},{line,142}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,226}]}]}
    ancestors: [<0.276.0>]
    message_queue_len: 1
    messages: [{'EXIT',<0.278.0>,normal}]
    links: [<0.276.0>,<0.44.0>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 610
    stack_size: 29
    reductions: 213
  neighbours:
2021-07-22 15:13:24 =SUPERVISOR REPORT====
     Supervisor: {local,'Elixir.Processor.Supervisor'}
     Context:    start_error
     Reason:     {already_started,<0.280.0>}
     Offender:   [{pid,undefined},{id,'Elixir.Trello'},{mfargs,{'Elixir.Processor',start_link,[<<"trello">>]}},{restart_type,permanent},{significant,false},{shutdown,5000},{child_type,worker}]

2021-07-22 15:13:24 =CRASH REPORT====
  crasher:
    initial call: application_master:init/4
    pid: <0.277.0>
    registered_name: []
    exception exit: {{{shutdown,{failed_to_start_child,'Elixir.Trello',{already_started,<0.280.0>}}},{'Elixir.Processor.Application',start,[normal,[]]}},[{application_master,init,4,[{file,"application_master.erl"},{line,142}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,226}]}]}
    ancestors: [<0.276.0>]
    message_queue_len: 1
    messages: [{'EXIT',<0.278.0>,normal}]
    links: [<0.276.0>,<0.44.0>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 610
    stack_size: 29
    reductions: 228
  neighbours:
2021-07-22 15:14:05 =SUPERVISOR REPORT====
     Supervisor: {local,'Elixir.Processor.Supervisor'}
     Context:    start_error
     Reason:     <0.280.0>
     Offender:   [{pid,undefined},{id,'Elixir.Git'},{mfargs,{'Elixir.Processor',start_link,[<<"github">>]}},{restart_type,permanent},{significant,false},{shutdown,5000},{child_type,worker}]

2021-07-22 15:14:05 =CRASH REPORT====
  crasher:
    initial call: application_master:init/4
    pid: <0.277.0>
    registered_name: []
    exception exit: {{{shutdown,{failed_to_start_child,'Elixir.Git',<0.280.0>}},{'Elixir.Processor.Application',start,[normal,[]]}},[{application_master,init,4,[{file,"application_master.erl"},{line,142}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,226}]}]}
    ancestors: [<0.276.0>]
    message_queue_len: 1
    messages: [{'EXIT',<0.278.0>,normal}]
    links: [<0.276.0>,<0.44.0>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 376
    stack_size: 29
    reductions: 198
  neighbours:
2021-07-22 15:14:24 =SUPERVISOR REPORT====
     Supervisor: {local,'Elixir.Processor.Supervisor'}
     Context:    start_error
     Reason:     {already_started,<0.280.0>}
     Offender:   [{pid,undefined},{id,'Elixir.Trello'},{mfargs,{'Elixir.Processor',start_link,[<<"trello">>]}},{restart_type,permanent},{significant,false},{shutdown,5000},{child_type,worker}]

2021-07-22 15:14:24 =CRASH REPORT====
  crasher:
    initial call: application_master:init/4
    pid: <0.277.0>
    registered_name: []
    exception exit: {{{shutdown,{failed_to_start_child,'Elixir.Trello',{already_started,<0.280.0>}}},{'Elixir.Processor.Application',start,[normal,[]]}},[{application_master,init,4,[{file,"application_master.erl"},{line,142}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,226}]}]}
    ancestors: [<0.276.0>]
    message_queue_len: 1
    messages: [{'EXIT',<0.278.0>,normal}]
    links: [<0.276.0>,<0.44.0>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 610
    stack_size: 29
    reductions: 208
  neighbours:
2021-07-22 15:15:17 =SUPERVISOR REPORT====
     Supervisor: {local,'Elixir.Processor.Supervisor'}
     Context:    start_error
     Reason:     {'EXIT',{#{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"expected :name option to be one of the following:\n\n  * nil\n  * atom\n  * {:global, term}\n  * {:via, module, term}\n\nGot: \"github\"\n">>},[{'Elixir.GenServer',do_start,4,[{file,"lib/gen_server.ex"},{line,944}]},{supervisor,do_start_child_i,3,[{file,"supervisor.erl"},{line,414}]},{supervisor,do_start_child,2,[{file,"supervisor.erl"},{line,400}]},{supervisor,'-start_children/2-fun-0-',3,[{file,"supervisor.erl"},{line,384}]},{supervisor,children_map,4,[{file,"supervisor.erl"},{line,1234}]},{supervisor,init_children,2,[{file,"supervisor.erl"},{line,350}]},{gen_server,init_it,2,[{file,"gen_server.erl"},{line,423}]},{gen_server,init_it,6,[{file,"gen_server.erl"},{line,390}]}]}}
     Offender:   [{pid,undefined},{id,'Elixir.Git'},{mfargs,{'Elixir.Processor',start_link,[<<"github">>]}},{restart_type,permanent},{significant,false},{shutdown,5000},{child_type,worker}]

2021-07-22 15:15:17 =CRASH REPORT====
  crasher:
    initial call: application_master:init/4
    pid: <0.277.0>
    registered_name: []
    exception exit: {{{shutdown,{failed_to_start_child,'Elixir.Git',{'EXIT',{#{'__exception__' => true,'__struct__' => 'Elixir.ArgumentError',message => <<"expected :name option to be one of the following:\n\n  * nil\n  * atom\n  * {:global, term}\n  * {:via, module, term}\n\nGot: \"github\"\n">>},[{'Elixir.GenServer',do_start,4,[{file,"lib/gen_server.ex"},{line,944}]},{supervisor,do_start_child_i,3,[{file,"supervisor.erl"},{line,414}]},{supervisor,do_start_child,2,[{file,"supervisor.erl"},{line,400}]},{supervisor,'-start_children/2-fun-0-',3,[{file,"supervisor.erl"},{line,384}]},{supervisor,children_map,4,[{file,"supervisor.erl"},{line,1234}]},{supervisor,init_children,2,[{file,"supervisor.erl"},{line,350}]},{gen_server,init_it,2,[{file,"gen_server.erl"},{line,423}]},{gen_server,init_it,6,[{file,"gen_server.erl"},{line,390}]}]}}}},{'Elixir.Processor.Application',start,[normal,[]]}},[{application_master,init,4,[{file,"application_master.erl"},{line,142}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,226}]}]}
    ancestors: [<0.276.0>]
    message_queue_len: 1
    messages: [{'EXIT',<0.278.0>,normal}]
    links: [<0.276.0>,<0.44.0>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 987
    stack_size: 29
    reductions: 227
  neighbours:
2021-07-22 15:15:30 =SUPERVISOR REPORT====
     Supervisor: {local,'Elixir.Processor.Supervisor'}
     Context:    start_error
     Reason:     {{shutdown,{server_initiated_close,405,<<"RESOURCE_LOCKED - cannot obtain exclusive access to locked queue 'raw_input' in vhost '/'. It could be originally declared on another connection or the exclusive property value does not match that of the original declaration.">>}},{gen_server,call,[<0.309.0>,{call,{'queue.declare',0,<<"raw_input">>,false,false,true,false,false,[]},none,<0.297.0>},60000]}}
     Offender:   [{pid,undefined},{id,'Elixir.Trello'},{mfargs,{'Elixir.Processor',start_link,[<<"trello">>]}},{restart_type,permanent},{significant,false},{shutdown,5000},{child_type,worker}]

2021-07-22 15:15:30 =CRASH REPORT====
  crasher:
    initial call: application_master:init/4
    pid: <0.277.0>
    registered_name: []
    exception exit: {{{shutdown,{failed_to_start_child,'Elixir.Trello',{{shutdown,{server_initiated_close,405,<<"RESOURCE_LOCKED - cannot obtain exclusive access to locked queue 'raw_input' in vhost '/'. It could be originally declared on another connection or the exclusive property value does not match that of the original declaration.">>}},{gen_server,call,[<0.309.0>,{call,{'queue.declare',0,<<"raw_input">>,false,false,true,false,false,[]},none,<0.297.0>},60000]}}}},{'Elixir.Processor.Application',start,[normal,[]]}},[{application_master,init,4,[{file,"application_master.erl"},{line,142}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,226}]}]}
    ancestors: [<0.276.0>]
    message_queue_len: 1
    messages: [{'EXIT',<0.278.0>,normal}]
    links: [<0.276.0>,<0.44.0>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 610
    stack_size: 29
    reductions: 213
  neighbours:
2021-07-22 15:17:21 =SUPERVISOR REPORT====
     Supervisor: {local,'Elixir.Processor.Supervisor'}
     Context:    start_error
     Reason:     {already_started,<0.280.0>}
     Offender:   [{pid,undefined},{id,'Elixir.Trello'},{mfargs,{'Elixir.Processor',start_link,[<<"trello">>]}},{restart_type,permanent},{significant,false},{shutdown,5000},{child_type,worker}]

2021-07-22 15:17:21 =CRASH REPORT====
  crasher:
    initial call: application_master:init/4
    pid: <0.277.0>
    registered_name: []
    exception exit: {{{shutdown,{failed_to_start_child,'Elixir.Trello',{already_started,<0.280.0>}}},{'Elixir.Processor.Application',start,[normal,[]]}},[{application_master,init,4,[{file,"application_master.erl"},{line,142}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,226}]}]}
    ancestors: [<0.276.0>]
    message_queue_len: 1
    messages: [{'EXIT',<0.278.0>,normal}]
    links: [<0.276.0>,<0.44.0>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 610
    stack_size: 29
    reductions: 208
  neighbours:
2021-07-22 15:17:34 =SUPERVISOR REPORT====
     Supervisor: {local,'Elixir.Processor.Supervisor'}
     Context:    start_error
     Reason:     {{shutdown,{server_initiated_close,405,<<"RESOURCE_LOCKED - cannot obtain exclusive access to locked queue 'raw_input' in vhost '/'. It could be originally declared on another connection or the exclusive property value does not match that of the original declaration.">>}},{gen_server,call,[<0.294.0>,{call,{'queue.declare',0,<<"raw_input">>,false,false,true,false,false,[]},none,<0.280.0>},60000]}}
     Offender:   [{pid,undefined},{id,'Elixir.Git'},{mfargs,{'Elixir.Processor',start_link,[<<"github">>]}},{restart_type,permanent},{significant,false},{shutdown,5000},{child_type,worker}]

2021-07-22 15:17:34 =CRASH REPORT====
  crasher:
    initial call: application_master:init/4
    pid: <0.277.0>
    registered_name: []
    exception exit: {{{shutdown,{failed_to_start_child,'Elixir.Git',{{shutdown,{server_initiated_close,405,<<"RESOURCE_LOCKED - cannot obtain exclusive access to locked queue 'raw_input' in vhost '/'. It could be originally declared on another connection or the exclusive property value does not match that of the original declaration.">>}},{gen_server,call,[<0.294.0>,{call,{'queue.declare',0,<<"raw_input">>,false,false,true,false,false,[]},none,<0.280.0>},60000]}}}},{'Elixir.Processor.Application',start,[normal,[]]}},[{application_master,init,4,[{file,"application_master.erl"},{line,142}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,226}]}]}
    ancestors: [<0.276.0>]
    message_queue_len: 1
    messages: [{'EXIT',<0.278.0>,normal}]
    links: [<0.276.0>,<0.44.0>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 376
    stack_size: 29
    reductions: 204
  neighbours:
2021-07-22 15:17:45 =SUPERVISOR REPORT====
     Supervisor: {local,'Elixir.Processor.Supervisor'}
     Context:    start_error
     Reason:     {{shutdown,{server_initiated_close,405,<<"RESOURCE_LOCKED - cannot obtain exclusive access to locked queue 'raw_input' in vhost '/'. It could be originally declared on another connection or the exclusive property value does not match that of the original declaration.">>}},{gen_server,call,[<0.279.0>,{call,{'queue.declare',0,<<"raw_input">>,false,false,true,false,false,[]},none,<0.265.0>},60000]}}
     Offender:   [{pid,undefined},{id,'Elixir.Git'},{mfargs,{'Elixir.Processor',start_link,[<<"github">>]}},{restart_type,permanent},{significant,false},{shutdown,5000},{child_type,worker}]

2021-07-22 15:17:45 =CRASH REPORT====
  crasher:
    initial call: application_master:init/4
    pid: <0.262.0>
    registered_name: []
    exception exit: {{{shutdown,{failed_to_start_child,'Elixir.Git',{{shutdown,{server_initiated_close,405,<<"RESOURCE_LOCKED - cannot obtain exclusive access to locked queue 'raw_input' in vhost '/'. It could be originally declared on another connection or the exclusive property value does not match that of the original declaration.">>}},{gen_server,call,[<0.279.0>,{call,{'queue.declare',0,<<"raw_input">>,false,false,true,false,false,[]},none,<0.265.0>},60000]}}}},{'Elixir.Processor.Application',start,[normal,[]]}},[{application_master,init,4,[{file,"application_master.erl"},{line,142}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,226}]}]}
    ancestors: [<0.261.0>]
    message_queue_len: 1
    messages: [{'EXIT',<0.263.0>,normal}]
    links: [<0.261.0>,<0.44.0>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 610
    stack_size: 29
    reductions: 203
  neighbours:
2021-07-22 15:18:08 =SUPERVISOR REPORT====
     Supervisor: {local,'Elixir.Processor.Supervisor'}
     Context:    start_error
     Reason:     {{shutdown,{server_initiated_close,405,<<"RESOURCE_LOCKED - cannot obtain exclusive access to locked queue 'raw_input' in vhost '/'. It could be originally declared on another connection or the exclusive property value does not match that of the original declaration.">>}},{gen_server,call,[<0.279.0>,{call,{'queue.declare',0,<<"raw_input">>,false,false,true,false,false,[]},none,<0.265.0>},60000]}}
     Offender:   [{pid,undefined},{id,'Elixir.Git'},{mfargs,{'Elixir.Processor',start_link,[<<"github">>]}},{restart_type,permanent},{significant,false},{shutdown,5000},{child_type,worker}]

2021-07-22 15:18:08 =CRASH REPORT====
  crasher:
    initial call: application_master:init/4
    pid: <0.262.0>
    registered_name: []
    exception exit: {{{shutdown,{failed_to_start_child,'Elixir.Git',{{shutdown,{server_initiated_close,405,<<"RESOURCE_LOCKED - cannot obtain exclusive access to locked queue 'raw_input' in vhost '/'. It could be originally declared on another connection or the exclusive property value does not match that of the original declaration.">>}},{gen_server,call,[<0.279.0>,{call,{'queue.declare',0,<<"raw_input">>,false,false,true,false,false,[]},none,<0.265.0>},60000]}}}},{'Elixir.Processor.Application',start,[normal,[]]}},[{application_master,init,4,[{file,"application_master.erl"},{line,142}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,226}]}]}
    ancestors: [<0.261.0>]
    message_queue_len: 1
    messages: [{'EXIT',<0.263.0>,normal}]
    links: [<0.261.0>,<0.44.0>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 376
    stack_size: 29
    reductions: 204
  neighbours:
2021-07-22 15:23:10 =SUPERVISOR REPORT====
     Supervisor: {local,'Elixir.Processor.Supervisor'}
     Context:    child_terminated
     Reason:     killed
     Offender:   [{pid,<0.265.0>},{id,'Elixir.Git'},{mfargs,{'Elixir.Processor',start_link,[<<"github">>]}},{restart_type,permanent},{significant,false},{shutdown,5000},{child_type,worker}]

2021-07-22 15:23:22 =SUPERVISOR REPORT====
     Supervisor: {local,'Elixir.Processor.Supervisor'}
     Context:    child_terminated
     Reason:     killed
     Offender:   [{pid,<0.335.0>},{id,'Elixir.Git'},{mfargs,{'Elixir.Processor',start_link,[<<"github">>]}},{restart_type,permanent},{significant,false},{shutdown,5000},{child_type,worker}]

2021-07-22 15:23:34 =SUPERVISOR REPORT====
     Supervisor: {local,'Elixir.Processor.Supervisor'}
     Context:    child_terminated
     Reason:     killed
     Offender:   [{pid,<0.282.0>},{id,'Elixir.Trello'},{mfargs,{'Elixir.Processor',start_link,[<<"trello">>]}},{restart_type,permanent},{significant,false},{shutdown,5000},{child_type,worker}]

2021-07-22 15:23:42 =CRASH REPORT====
  crasher:
    initial call: application_master:init/4
    pid: <0.262.0>
    registered_name: []
    exception exit: {killed,[{application_master,terminate,2,[{file,"application_master.erl"},{line,236}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,226}]}]}
    ancestors: [<0.261.0>]
    message_queue_len: 0
    messages: []
    links: [<0.44.0>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 1598
    stack_size: 29
    reductions: 11278
  neighbours:
2021-07-22 15:33:54 =SUPERVISOR REPORT====
     Supervisor: {local,'Elixir.Processor.Supervisor'}
     Context:    child_terminated
     Reason:     killed
     Offender:   [{pid,<0.271.0>},{id,'Elixir.Trello'},{mfargs,{'Elixir.Processor',start_link,[<<"trello">>]}},{restart_type,permanent},{significant,false},{shutdown,5000},{child_type,worker}]

2021-07-22 15:33:54 =SUPERVISOR REPORT====
     Supervisor: {local,'Elixir.Processor.Supervisor'}
     Context:    child_terminated
     Reason:     killed
     Offender:   [{pid,<0.254.0>},{id,'Elixir.Git'},{mfargs,{'Elixir.Processor',start_link,[<<"github">>]}},{restart_type,permanent},{significant,false},{shutdown,5000},{child_type,worker}]

